lib/BuildSystem/BuildSystemFrontend.cpp

```diff
@@ -232,12 +232,21 @@ class BuildSystemFrontendExecutionQueueDelegate
 };
 
 struct BuildSystemFrontendDelegateImpl {
+
+  /// The status of delegate.
+  enum class Status {
+    Uninitialized = 0,
```

Comment:
debug mode: assertional failure
release mode: not sure, will this still return false? https://github.com/apple/swift-llbuild/blob/master/lib/BuildSystem/BuildSystemFrontend.cpp#L544